homeassistant/components/system_bridge/__init__.py

```diff
@@ -186,6 +196,50 @@ def valid_device(device: str) -> str:
                 raise vol.Invalid from exception
         raise vol.Invalid(f"Device {device} does not exist")
 
+    async def handle_get_process_by_id(call: ServiceCall) -> ServiceResponse:
+        """Handle the get process by id service call."""
+        _LOGGER.info("Get process by id: %s", call.data)
```

Comment:
We dislike to log on info, but I see that you do this on more places, can you maybe fix this in a followup?

Reply:
Will do, I've changed the ones here to debug level

Reply:
https://github.com/home-assistant/core/pull/112175